src/dotnet-razor-tooling/Internal/ResolveTagHelpersCommand.cs

```diff
@@ -82,6 +74,23 @@ internal static void Register(CommandLineApplication app)
             });
         }
 
+        /// <summary>
+        /// Internal for testing.
+        /// </summary>
+        public static ProjectContext ResolveProjectContext(string projectFilePath)
```

Comment:
Ok. Maybe remove the comment in that case entirely in that case. No harm in it being public